docs/_user_guide/julia.md

```diff
@@ -0,0 +1,109 @@
+# Running Julia Interactively
+
+On the Yens, Julia is available via the `module` command. 
+
+See all currently installed versions with:
+
+```title="Terminal Input"
+module avail julia
+```
+
+You will see different Julia versions listed:
+
+```{.yaml .no-copy title="Terminal Output"}
+
+-------------------------------------------- Global Aliases --------------------------------------------
+
+
+---------------------------------------- /software/modules/Core ----------------------------------------
+   julia/0.7.0    julia/1.0.2    julia/1.3.1    julia/1.6.2    julia/1.8.0    julia/1.10.2 (D)
+   julia/1.0.0    julia/1.2.0    julia/1.5.1    julia/1.7.3    julia/1.9.2
+
+  Where:
+   D:  Default Module
+
+If the avail list is too long consider trying:
```

Comment:
Perhaps cut off the boilerplate output after "D: Default Module"

Reply:
done